src/main/resources/log4j2.xml

```diff
@@ -1,13 +1,38 @@
-<Configuration status="WARN">
-    <Appenders>
-        <Console name="STDOUT" target="SYSTEM_OUT">
-            <PatternLayout pattern="[%d] %-5level - %msg (%F:%L)%n"/>
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
+  ~ /*
+  ~  *  Copyright 2024 The Hyve
+  ~  *
+  ~  *  Licensed under the Apache License, Version 2.0 (the "License");
+  ~  *  you may not use this file except in compliance with the License.
+  ~  *  You may obtain a copy of the License at
+  ~  *
+  ~  *    http://www.apache.org/licenses/LICENSE-2.0
+  ~  *
+  ~  *  Unless required by applicable law or agreed to in writing, software
+  ~  *  distributed under the License is distributed on an "AS IS" BASIS,
+  ~  *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~  *  See the License for the specific language governing permissions and
+  ~  *  limitations under the License.
+  ~  */
+  -->
+<configuration status="INFO">
+    <appenders>
+        <Console name="Console" target="SYSTEM_OUT">
+            <PatternLayout
+                    pattern="%d{HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n"
+            />
         </Console>
-    </Appenders>
+        <!-- For Sentry to work the DSN must be set via SENTRY_DSN environment variable
+             When SENTRY_DSN is empty string, the Sentry SDK is disabled -->
+        <Sentry name="Sentry" debug="false"/>
+    </appenders>
 
-    <Loggers>
-        <Root level="${env:LOG4J_LOG_LEVEL:-INFO}">
-            <AppenderRef ref="STDOUT"/>
-        </Root>
-    </Loggers>
-</Configuration>
+    <loggers>
+        <root level="INFO">
+            <appender-ref ref="Console" />
```

Comment:
could console log level still be configurable (as it was with env.LOG4J_LOG_LEVEL)?

Reply:
@ewelinagr I think this is still possible yes. If I am correct, the sentry log level is independent from the root logger level (see [here](https://docs.sentry.io/platforms/java/legacy/log4j/#usage))

Reply:
@ewelinagr Indeed the env.LOG4J_LOG_LEVEL is removed in this PR. I will reinstate this in a next commit.